llvm/include/llvm/ADT/APFloat.h

```diff
@@ -1483,6 +1483,19 @@ inline APFloat minimum(const APFloat &A, const APFloat &B) {
   return B < A ? B : A;
 }
 
+/// Implements IEEE 754-2019 minimumNumber semantics. Returns the smaller
+/// of 2 arguments, not propagating NaNs and treating -0 as less than +0.
+LLVM_READONLY
+inline APFloat minimumnum(const APFloat &A, const APFloat &B) {
+  if (A.isNaN())
+    return B.isNaN() ? B.makeQuiet() : B;
+  if (B.isNaN())
```

Comment:
Doesn't handle signaling case correctly?

Reply:
Maybe I don't understand your question well.
If you mean about is the result correct if either/both of A or/and B is/are sNaN, I think that it can give correct result.
When we in `if (B.isNaN())`, we are sure that `A` is not NaN.

If you are talking about exception handling, in fact I am not sure.  I just noticed that `minnum`/`minimum` don't do
very special work for exception handling.

Reply:
I thought these had the IEEE behavior and returning a quiet nan for any signaling input, I see now they ignore the signalingness like the common buggy fmin case

Reply:
Yes. It is new in IEEE754-2019.  I guess what you talked about is minNum/maxNum in IEEE754-2008.
minNum/maxNum have disappeared from IEEE754-2019.

Maybe this link is useful: https://www.gnu.org/software/libc/manual/html_node/Misc-FP-Arithmetic.html

Reply:
So my plan is to map llvm.minnum/llvm.maxnum to fmin/fmax, and 
llvm.minimumnum/llvm.maximumnum is mapped to fminimum_num/fmaximum_num